app/main.py

```diff
@@ -1 +1,26 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is a class variable, which is appropriate for tracking all instances of `Animal`. Ensure that this list is managed carefully to avoid potential issues with object references, especially when instances are deleted or go out of scope.